config/initializers/cors.rb

```diff
@@ -1,10 +1,3 @@
-# Be sure to restart your server when you modify this file.
-
-# Avoid CORS issues when API is called from the frontend app.
-# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
-
-# Read more: https://github.com/cyu/rack-cors
-
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
```

Comment:
funny that you removed that commit because just yesterday while I was having cors issues with my ruby backend it saved my life. 
![thinkmeme](https://user-images.githubusercontent.com/45549491/67035144-1449c000-f0e7-11e9-8ea4-9446684a459c.jpg)